pandas/util/_decorators.py

```diff
@@ -314,14 +314,15 @@ def indent(text, indents=1):
 
 def make_signature(func):
     """
-    Returns a string repr of the arg list of a func call, with any defaults.
+    Returns a string repr of tuple - the arg list of a func call with
```

Comment:
Short description for docstrings should fit on one line

Reply:
Done